lib/tasks/templates_rendering_statuses.rake

```diff
@@ -0,0 +1,24 @@
+# TRANSLATORS: do not translate
+desc <<~END_DESC
+  Refresh templates rendering statuses for all managed hosts
+
+    Example:
+      rake templates_rendering_statuses:refresh
+
+END_DESC
+
+namespace :templates_rendering_statuses do
+  task refresh: :environment do
+    Host::Managed.where(managed: true).in_batches(of: 100) do |batch|
+      batch.map do |host|
+        puts "#{host} - Refreshing templates rendering status"
+        host.refresh_statuses([HostStatus::TemplatesRenderingStatus])
+        puts Rainbow("#{host} - Successfully refreshed templates rendering status").green
+      rescue StandardError => e
+        puts Rainbow("#{host} - Error while refreshing templates rendering status: #{e}").red
+        puts Rainbow(e.backtrace.join("\n"))
+        nil
+      end
+    end
+  end
+end
```

Comment:
Can you make sure this is not executed as root? In that case exit immediately, rendering any template as root would be a huge security hole even in safemode. Just in case someone would configure this or start this from command line as root We do run rake tasks via `foreman` account on production deployments via `foreman-rake`, this is just an extra security feature.

Reply:
would checking the user name and UID be good enough? 
```ruby
Etc.getpwuid.name != 'root' && Etc.getpwuid.uid != 0
```

Reply:
The uid should be enough.